boards/arm/nucleo_h723zg/nucleo_h723zg.dts

```diff
@@ -95,6 +95,11 @@
 	status = "okay";
 };
 
+&usart2 {
+	pinctrl-0 = <&usart2_tx_pd5 &usart2_rx_pd6>;
+	current-speed = <115200>;
+};
```

Comment:
Missing:
`status = "okay";`

Reply:
It is enabled in the testcase (overlay), not to impact other applications.

Reply:
changed, now enabling in the nulceo_h723zg board